saw-script/src/SAWScript/Interpreter.hs

```diff
@@ -5933,6 +5933,13 @@ primitives = Map.fromList $
     , "be found in the MIRModule, this will raise an error."
     ]
 
+  , prim "mir_find_name" "MIRModule -> String -> [MIRType] -> String"
+    (funVal3 mir_find_name)
+    Experimental
+    [ "Find the name of a polymorphic function."
+    , "Throws an error if the name is unknown or ambiguous."
+    ]
```

Comment:
I think saying "find the name of a polymorphic function" is a bit misleading, since you're not intending to the return the name of the polymorphic function, but rather an _instantiation_ of the polymorphic function, under a different name. To clarify this, it would be helpful to adopt some of the verbiage from `mir_find_adt`'s help text, which describes the role of the `[MIRType]` arguments. Something like this, perhaps:


```suggestion
    [ "Consult the given MIRModule to find an instantiation of a function"
    , "with the given String as an identifier and the given MIRTypes as the"
    , "types used to instantiate the type parameters. If such an instantiation cannot"
    , "be found in the MIRModule, this will raise an error."
    ]
```